tests/functional/restart/bad-job-host/flow.cylc

```diff
@@ -1,7 +1,6 @@
 #!jinja2
 [cylc]
     [[events]]
-        abort if any task fails = True
```

Comment:
I can't see the matching test file change for this, is it one of those convoluted ones?